helm-charts/azure-api-management-gateway/templates/ingress-stable.yaml

```diff
@@ -38,7 +39,11 @@ spec:
               service:
                 name: {{ $fullName }}
                 port:
+                {{- if $host.useHttpsBackend }}
```

Comment:
Can you add this to `values.yml` with a default of false please? (to not break things)

Reply:
sure ✅